### ISSUE TITLE
Possible bug with Marker copy

### ISSUE THREAD

**Issue by ayshon** [data-8/datascience#566]
### Problem Summary:

While trying to create a unit test for the Marker.copy method I found that Marker.copy does not return a copy of the Marker, but instead it throws the following error:

```
>>> marker2 = marker1.copy()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/llerasea/datascience/datascience/maps.py", line 526, in copy
    return type(self)(self.lat_lon[:], **self._attrs)
TypeError: Marker.__init__() missing 1 required positional argument: 'lon'
```

### Expected Behavior

Marker.copy returns a copy of the marker.

### Actual Behavior

Throws the following error:
```
>>> marker2 = marker1.copy()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/llerasea/datascience/datascience/maps.py", line 526, in copy
    return type(self)(self.lat_lon[:], **self._attrs)
TypeError: Marker.__init__() missing 1 required positional argument: 'lon'
```
Screenshot of actual behavior:
![image](https://user-images.githubusercontent.com/60166218/206087731-5beb54a5-195b-4afa-90a6-044d430c4716.png)

### Possible Bug

Unless I am calling the copy method incorrectly, I believe there is a bug with the copy method of the Marker class.

### Proposed Fix

Currently the copy method is written as the following:

```
def copy(self):
        """Return a deep copy"""
        return type(self)(self.lat_lon[:], **self._attrs)
```

I found that the issue is resolved by separating lat and lon in self.lat_lon. For example:

```
def copy(self):
        """Return a deep copy"""
        return type(self)(self.lat_lon[0], self.lat_lon[1], **self._attrs)
```

**Comment by adnanhemani** [comment]
Hi, yes, you've discovered a bug! When calling `[:]` on a Python tuple, you end up just getting the tuple and that will pass in as a single argument to the class initializer at this point. Let me create a CR to fix this bug.

**Comment by davidwagner** [comment]
Fixed in #571
